src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -24,7 +23,7 @@
     }
 
     static Validation<String, String> isAboutToSubmit(String eventId) {
-        return ABOUT_TO_SUBMIT.equals(eventId)
+        return ATTACH_TO_EXISTING_CASE.equals(eventId)
```

Comment:
shouldn't method name be updated as well??

Reply:
sure missed that.